2-mandatory/4-stocks.js

```diff
@@ -48,7 +53,13 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+    return closingPricesForAllStocks
+        .map(prices => {
+            const day5price = prices.pop();
+            const day1price = prices.shift();
```

Comment:
with pop() and shift() methods you remove elements from original array. It affects the following functions and tests.

Reply:
Great feedback!